packages/@react-spectrum/dialog/chromatic/DialogLanguages.chromatic.tsx

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright 2023 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import {ActionButton, Button} from '@react-spectrum/button';
+import {ButtonGroup} from '@react-spectrum/buttongroup';
+import {Cell, Column, Row, TableBody, TableHeader, TableView} from '@react-spectrum/table';
+import {Content, Header} from '@react-spectrum/view';
+import {Dialog, DialogTrigger} from '../';
+import {Divider} from '@react-spectrum/divider';
+import {Heading} from '@react-spectrum/text';
+// @ts-ignore
+import intlMessages from './intl/*.json';
+import {Item, TagGroup} from '@react-spectrum/tag';
+import {Provider} from '@react-spectrum/Provider';
+import React from 'react';
+import {useLocalizedStringFormatter} from '@react-aria/i18n';
+
+export default {
+  title: 'Languages/Dialog',
+  parameters: {
+    chromaticProvider: {
+      colorSchemes: ['darkest'],
+      express: false,
+      locales: ['en-US'],
+      scales: ['medium']
+    }
+  }
+};
+
+export let TranslateDialog = (dialogProps) => {
```

Comment:
I wrote these in Chromatic and imported them into Storybook. I thought about moving them to Storybook, but it would add more imports to those files that are already big, while here they are contained in these files and not causing noise.